### PR TITLE
Add continue and limit to custom objects spec

### PR DESCRIPTION
**Pull request by scottilee**
Adds continue and limit parameters to the custom objects specification.

Fixes https://github.com/kubernetes-client/python/issues/965.

### CODE REVIEW

File: openapi/custom_objects_spec.json [hunk 1]
```diff
@@ -352,13 +366,27 @@
...
+          "in": "query"
+        },
```
Comment (roycaihw) [inline openapi/custom_objects_spec.json]: This is a delete operation, which doesn't support paging. Please move this parameter to `listNamespacedCustomObject`. Same for the `limit` parameter in L383-L389

Comment (scottilee) [reply openapi/custom_objects_spec.json]: Fixed.

**Comment by brendandburns** [comment]
LGTM, @roycaihw any final comments before we merge?

**Comment by roycaihw** [comment]
/lgtm
/approve

Thanks

**Comment by roycaihw** [comment]
Manually tagged based on the consensus in the thread.

**Pull request merged**